lwallet/cardano-sl-lwallet.cabal

```diff
@@ -0,0 +1,121 @@
+name:                cardano-sl-lwallet
+version:             0.4.4
+synopsis:            Cardano SL - Light wallet
+description:         Cardano SL - Light wallet
+license:             MIT
+license-file:        LICENSE
+author:              Serokell
+maintainer:          hi@serokell.io
+copyright:           2016 IOHK
+category:            Currency
+build-type:          Simple
+cabal-version:       >=1.10
+
+executable cardano-wallet
+  hs-source-dirs:      .
+  main-is:             Main.hs
+  other-modules:
+                       Command
+                       WalletOptions
+
+                       Pos.Wallet.Light
+                       Pos.Wallet.Light.Launcher
+                       Pos.Wallet.Light.Launcher.Runner
+                       Pos.Wallet.Light.Launcher.Param
+                       Pos.Wallet.Light.Mode
+                       Pos.Wallet.Light.State
+                       Pos.Wallet.Light.State.Acidic
+                       Pos.Wallet.Light.State.Core
+                       Pos.Wallet.Light.State.State
+                       Pos.Wallet.Light.State.Storage
+                       Pos.Wallet.Light.State.Storage.Block
+                       Pos.Wallet.Light.State.Storage.Tx
+                       Pos.Wallet.Light.Redirect
+                       Pos.Wallet.Light.Web
+                       Pos.Wallet.Light.Web.Server
+
+  -- TODO: what to do?
+  -- if !flag(for-installer)
```

Comment:
I think this todo can be removed because the travis/appveyor scripts already do the right thing.